vivisect/__init__.py

```diff
@@ -1933,14 +1933,16 @@ def analyzePointer(self, va):
         no idea.
         """
         if self.getLocation(va) is not None:
-            return None
+            return (None, None)
```

Comment:
I think you missed a spot where analyzePointer is called (line 1187 on master).

Reply:
done